src/aggregation/bucket/term_agg.rs

```diff
@@ -544,6 +548,27 @@ impl SegmentTermCollector {
                 let val = bool::from_u64(val);
                 dict.insert(IntermediateKey::Bool(val), intermediate_entry);
             }
+        } else if self.column_type == ColumnType::IpAddr {
+            let arc_inner = agg_with_accessor
```

Comment:
let's put a decent name and a type on this. `arc_inner` is not helpful.
```suggestion
            let arc_inner = agg_with_accessor
```

Reply:
done